lib/build.gradle.kts

```diff
@@ -35,6 +36,17 @@ android {
     }
 }
 
+publishing {
+    publications {
+        create<MavenPublication>("Maven") {
```

Comment:
Where does this publish to? Have you set that up?

Reply:
This should be used to publish to JitPack, I'll set up maven myself later